### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Dinesh-ramisetti**

### CODE REVIEW

**Review by Botarun143 — CHANGES_REQUESTED** [review]
plz dont ask me anything

**Review by Botarun143 — CHANGES_REQUESTED** [review]
learn coding

**Comment by Dinesh-ramisetti** [comment]
>

> learn coding

ok bro

File: .idea/og.txt
Comment (Botarun143) [inline .idea/og.txt:1]: lern basics of java

**Review by Botarun143 — CHANGES_REQUESTED** [review]

File: .idea/og.txt
Comment (Dinesh-ramisetti) [reply .idea/og.txt:1]: ok bro

Comment (Botarun143) [reply .idea/og.txt:1]: do it fast

**Review by Botarun143 — CHANGES_REQUESTED** [review]

File: .idea/og.txt
Comment (Dinesh-ramisetti) [reply .idea/og.txt:1]: wait a minute bro i will make necessary changes as u said

Comment (Botarun143) [reply .idea/og.txt:1]: Ok

**Review by Botarun143 — APPROVED** [review]

File: .idea/og.txt
Comment (Botarun143) [reply .idea/og.txt:1]: i will approval

**Review by Botarun143 — APPROVED** [review]

File: .idea/og.txt
Comment (Nani0218) [inline .idea/og.txt:1]: explain this code

Comment (Dinesh-ramisetti) [reply .idea/og.txt:1]: ok after lunch break i will explain

**Pull request merged**